mvc/controllers/getReviewObject.js

```diff
@@ -0,0 +1,11 @@
+const fetchReviewObjectById = require("../models/fetchReviewObject.js");
+
+const getReviewObjectById = async (req, res, next) => {
+  const response = await fetchReviewObjectById(
+    req.params.review_id,
+    next
+  ).catch(next);
+  return res.status(200).send(response);
```

Comment:
This should be sent back on an object with a key of review and a value of the response, please.

Reply:
done